cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -153,6 +170,57 @@ function ManagementCredentials(props: Partial<StepProps>) {
         }
     };
 
+    useEffect(() => {
+        retrieveOsImages(awsState[STORE_SECTION_FORM].REGION);
+    }, [awsState[STORE_SECTION_FORM].REGION]);
+
+    function retrieveOsImages(region: string | undefined) {
```

Comment:
It might make things easier to remove `undefined` as a possible parameter. Alternatively, we could check it's value here before calling `AwsService.getAwsosImages()`

Reply:
Gonna remove `undefined` and check if `region` is a string